question.php

```diff
@@ -168,6 +168,20 @@ public function update_attempt_state_data_for_new_version(
 
         return $startdata;
     }
+
+    /**
+     * Check whether question has any specific feedback.
+     *
+     * @return bool
+     */
+    public function has_specific_feedback_for_row(): bool {
```

Comment:
Please we rename this to "has_specific_feedback". I think that is clearer.

Reply:
Hi @timhunt, thanks for the review. I have renamed has_specific_feedback_for_row() and pushed the patch.